bin/embedding-sdk/fixup-types-after-compilation.js

```diff
@@ -73,4 +73,9 @@ const fixupTypesAfterCompilation = () => {
   dtsFilePaths.forEach(replaceAliasedImports);
 };
 
+// if running as standalone script
```

Comment:
Thinking twice about this, I can make this script accept --watch and remove it from the webpack config